modules/network/pre-existing-vpc/README.md

```diff
@@ -16,14 +21,19 @@ sharing a single network module between deployment groups.
   source: modules/network/pre-existing-vpc
   settings:
   - project_id: $(vars.project_id)
```

Comment:
This probably shouldn't be a list, right?

Reply:
Good catch. That was in original commit but I didn't notice.